zerver/lib/bugdown/__init__.py

```diff
@@ -754,11 +754,14 @@ def handleMatch(self, match):
 def make_emoji(codepoint, display_string):
     # type: (Text, Text) -> Element
     src = '/static/generated/emoji/images/emoji/unicode/%s.png' % (codepoint,)
+    # Replace underscore in emoji's title with space
+    title = display_string[1:-1].replace("_", " ")
+
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
```

Comment:
i do feel like most sources i've come across are against the use of title (because it's usually not read by screen readers, and the screen readers that do use it will end up reading the alt & title)
the sources i'm talking about:
http://www.mediacurrent.com/blog/dont-rely-title-attribute-accessibility-seo
https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/
https://www.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/